### ISSUE TITLE
Add files directly in the note

### ISSUE THREAD

**Issue by collector-ynh** [TriliumNext/Trilium#3614]
### Trilium Version

0.58.8

### What operating system are you using?

Other Linux

### What is your setup?

Local (no sync)

### Operating System Version

Linux-Mint 21.1

### Description

When you want to add files to the note, the file opens as a separate note! I know that it is possible to integrate a note inside another one, but I don't want the files to create their own notes in the tree (images don't do that and integrate perfectly!), or hide the notes from the files and integrate them into the main note.

Is it possible to prevent files from creating their own notes? 
Is it possible to hide file notes?

**Comment by zadam** [comment]
> Is it possible to prevent files from creating their own notes?

No. Trilium's design is "everything is a note". Even images are notes, but due to popular demand they are hidden by default from the tree:

<img width="289" alt="image" src="https://user-images.githubusercontent.com/617641/219042429-9c39d4b2-794c-494f-9ec6-2be866360103.png">

**Comment by collector-ynh** [comment]
@zadam Thanks for the information

I found a roundabout way to hide the notes from the files, first I integrate the files on the main note and then I give them the label "#archived" while activating the "Hide archived notes" function!

I know this is not the best way to do it, but I found that this method for now 

I would really like to see a feature to hide notes from files in Trilium!

**Comment by eliandoran** [comment]
Now supported via attachments.
